test/parallel/test-process-hrtime.js

```diff
@@ -24,3 +24,9 @@ function validateTuple(tuple) {
     assert(isFinite(v));
   });
 }
+
+const base = process.hrtime();
+for (let i = 0; i < 1e5; i += 1) {
```

Comment:
Because it properly decremented the seconds for any uint32 ns passed previous to my change. Therefore it should probably continue to work?